### PR TITLE
chore: added arbitrum and arbitrumGoerli

### PR DESCRIPTION
**Pull request by heueristik**
Added networks supported by Aragon OSx

Tasks: [OS-804](https://aragonassociation.atlassian.net/browse/OS-804), [OS-811](https://aragonassociation.atlassian.net/browse/OS-811)

[OS-804]: https://aragonassociation.atlassian.net/browse/OS-804?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[OS-811]: https://aragonassociation.atlassian.net/browse/OS-811?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by heueristik** [comment]
Lint is failing for JS client. Do we want to skip linting it for now until tsdx is removed in https://aragonassociation.atlassian.net/browse/OS-817 @mathewmeconry?

File: packages/contracts/hardhat.config.ts [hunk 1]
```diff
@@ -104,6 +114,8 @@ const config: HardhatUserConfig = {
       polygonMumbai: process.env.POLYGONSCAN_API_KEY || '',
       base: process.env.BASESCAN_API_KEY || '',
       baseGoerli: process.env.BASESCAN_API_KEY || '',
+      arbitrum: process.env.ARBISCAN_API_KEY || '',
```
Comment (heueristik) [inline packages/contracts/hardhat.config.ts]:
```suggestion
      arbitrumOne: process.env.ARBISCAN_API_KEY || '',
```

File: packages/contracts/hardhat.config.ts [hunk 2]
```diff
@@ -104,6 +114,8 @@ const config: HardhatUserConfig = {
       polygonMumbai: process.env.POLYGONSCAN_API_KEY || '',
       base: process.env.BASESCAN_API_KEY || '',
       baseGoerli: process.env.BASESCAN_API_KEY || '',
+      arbitrumOne: process.env.ARBISCAN_API_KEY || '',
```
Comment (heueristik) [inline packages/contracts/hardhat.config.ts:117]: I double-checked that these are correct: https://github.com/NomicFoundation/hardhat/blob/f0ca6c8ff7ec48031ffaf368212b8dbe5bf5587e/packages/hardhat-verify/src/internal/chain-config.ts#L174

**Comment by mathewmeconry** [comment]
> Lint is failing for JS client. Do we want to skip linting it for now until tsdx is removed in https://aragonassociation.atlassian.net/browse/OS-817 @mathewmeconry?

sounds like a plan to me. do you wanna exclude it in this pull?

File: .github/workflows/formatting-linting.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ name: 'formatting & linting'
 on:
   workflow_dispatch:
   workflow_call:
+  push:
```
Comment (mathewmeconry) [inline .github/workflows/formatting-linting.yml:6]: Nice catch. Added https://aragonassociation.atlassian.net/browse/OS-846 to our task list to further improve it

**Review by mathewmeconry — APPROVED** [review]

File: .github/workflows/formatting-linting.yml [hunk 4]
```diff
@@ -29,9 +30,6 @@ jobs:
       - name: 'Lint contracts ts and js'
         run: yarn lint:contracts:ts >> $GITHUB_STEP_SUMMARY
 
-      - name: 'Lint contracts-ethers ts and js'
-        run: yarn lint:contracts-ethers:ts >> $GITHUB_STEP_SUMMARY
-
       - name: 'Lint js-client ts and js'
         run: yarn lint:js-client:ts >> $GITHUB_STEP_SUMMARY
```
Comment (heueristik) [inline .github/workflows/formatting-linting.yml]:
```suggestion
      #- name: 'Lint js-client ts and js'
      #  run: yarn lint:js-client:ts >> $GITHUB_STEP_SUMMARY
```

**Pull request merged**